docs/en/maintain/monitoring.md

```diff
@@ -2,14 +2,47 @@
 
 ## Overview
 
-
 The monitoring scheme of OpenMLDB is outlined as follows：
 
 - Use [Prometheus](https://prometheus.io) to collect monitoring metrics, [Grafana](https://grafana.com/oss/grafana/) to visualize metrics
 - OpenMLDB exporter exposes database-level metrics
 - Each component as a server itself expose component-level metrics
 - Uses [node_exporter](https://github.com/prometheus/node_exporter) to expose machine and operating system related metrics
 
+## Quick Deployment
+
+1. [Optional] Deploy node_exporter on each machine of OpenMLDB. If not deployed, it will not affect the display of Grafana OpenMLDB Dashboard.
+2. [Optional] Deploy an OpenMLDB exporter. If not deployed, it will only result in a few missing data in the Grafana OpenMLDB Dashboard, without affecting the monitoring of read and write operations.
+3. Start Prometheus with the following minimal configuration file. Fill in the corresponding IP addresses, but do not fill in the IP address of TaskManager (which does not support metrics):
+```yaml
+global:
+  scrape_interval:     15s # By default, scrape targets every 15 seconds.
+
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: openmldb_components
+    metrics_path: /brpc_metrics
+    static_configs:
+      - targets:
+        - nameserver_ip
+        - tablet_ip
+        - tablet_ip
+        - apiserver_ip
+```
+For complete configuration, refer to [openmldb_mixin/prometheus_example.yml](https://github.com/4paradigm/openmldb-exporter/blob/main/openmldb_mixin/prometheus_example.yml).
+
+Command reference: `docker run -d -v <config_file>:/etc/prometheus/prometheus.yml -p 9090:9090 -name promethues prom/prometheus`
+
+4. Start Grafana and use the OpenMLDB Dashboard template
+
+Command reference: `docker run -d -p 3000:3000 --name=grafana grafana/grafana-enterprise`
```

Comment:
grafana-oss

Reply:
https://grafana.com/docs/grafana/latest/setup-grafana/installation/docker/ enterprise is free too, just a reference, any problem?

Reply:
use oss